labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -88,7 +88,7 @@ object ListManipulationExercise01 {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList.filter((A) => A%2 == 1)
   }
 
   /**
```

Comment:
The one liners / built in functions are a huge time save haha. I used all the built in functions for this part, which was probably why I had a hard time understanding pattern matching later one in the assignment.